File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.js
class Solution {
    /**
     * @param {number[]} numbers
     * @param {number} target
     * @return {number[]}
     */
    twoSum(numbers, target) {
        let point1 = 0
        let point2 = numbers.length - 1

        while (target !== numbers[point1] + numbers[point2]) {
            if (target > numbers[point1] + numbers[point2]) {
                point1 += 1;
            } else {
                point2 -= 1;
            }
        }

        return [point1 + 1, point2 + 1]
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(len(nums)):
            for j in range(i + 1, len(nums)):
                totalSum = nums[i] + nums[j]
                if totalSum == target:
                    return [i, j]
        return [1453]